apps/src/templates/UnsafeRenderedMarkdown.jsx

```diff
@@ -37,7 +37,18 @@ export default class UnsafeRenderedMarkdown extends React.Component {
     const parser = this.props.openExternalLinksInNewTab
       ? extendedParser
       : remarkParser;
-    const processedMarkdown = parser.sourceToHtml(this.props.markdown);
+    let processedMarkdown = parser.sourceToHtml(this.props.markdown);
+
+    if (this.props.openExternalLinksInNewTab) {
+      processedMarkdown = processedMarkdown.replace(
+        /^<p>/,
+        '<p style="display: inline;">'
+      );
```

Comment:
at this point, we are no longer actually rendering markdown, we're rendering something else that looks a lot like markdown but isn't.

If we want to end up with results that look like this, we should do it in a separate component that makes it clear we're doing something heavily customized

Reply:
That's what we had before, but using marked.js instead of remark.js. Still worth switching from marked.js -> remark.js, or should I abandon PR https://github.com/code-dot-org/code-dot-org/pull/27841 entirely?